cloudslang-entities/src/main/java/io/cloudslang/lang/entities/encryption/EncryptionProvider.java

```diff
@@ -34,21 +34,25 @@ public static Encryption get() {
     }
 
     private static Encryption create() {
-        ApplicationContext applicationContext = ApplicationContextProvider.getApplicationContext();
-        if (applicationContext != null) {
-            Map<String, Encryption> encryptorMap = applicationContext.getBeansOfType(Encryption.class);
-            Encryption[] encryptors = encryptorMap.values().toArray(new Encryption[encryptorMap.size()]);
-            if (encryptors.length == 0) {
-                throw new RuntimeException("No encryptors found");
-            } else if (encryptors.length == 1) {
-                return encryptors[0];
-            } else if (encryptors.length == 2) {
-                return encryptors[0] instanceof DummyEncryptor ? encryptors[1] : encryptors[0];
-            } else {
-                throw new RuntimeException("Too many (" + encryptors.length + ") encryptors found");
+        Encryption[] encryptors;
+        try {
+            encryptors = SlangEntitiesSpringConfig.getEncryptors();
+        } catch (Exception theGivenEx) {
+            if (!StringUtils.containsIgnoreCase(theGivenEx.getMessage(), "ApplicationContextProvider bean missing")) {
```

Comment:
I don't like negations as well, but in this case it is expressing the intention of the code 👍